_posts/2016/02/2016-02-03-swdcretreat2015-university_course.html

```diff
@@ -0,0 +1,437 @@
+---
+layout: blog
+root: ../../..
+author: Daniel Chen
+title: "SWDCRetreat2015 - Round-table discussion: Software Carpentry as a University Course Recap"
+date: 2015-11-16
+time: "11:30:00"
+category: ["Community", "Teaching", "SWDCRetreat"]
+---
+<!-- start excerpt -->
+The <a href='http://swcarpentry.github.io/instructor-retreat-2015/'> inaugural Software Carpentry + Data Carpentry Instructor & Helper Retreat</a> is over!
+It was a long day, packed with tutorials, demos, and discussions.
+<a href='https://twitter.com/chendaniely'>I (Daniel Chen)</a>
+led a round table discussion with
+<a href='https://twitter.com/TiffanyTimbers'>Tiffany Timbers</a> and
+<a href='https://twitter.com/JennyBryan'>Jenny Bryan</a>.
+You can <a href='https://plus.google.com/events/c41eaub933br0odpb385rcg9j00'>watch</a> the discussion and/or
+<a href='http://pad.software-carpentry.org/swc-instructor-retreat-2015-7PM-UTC'>read</a> the notes on etherpad.
+
+<!-- end excerpt -->
+
+First, thanks to everyone who joined the discussion:
+<ul>
+  <li>Tiffany Timbers / @TiffanyTimbers</li>
+  <li>Jenny Bryan / @JennyBryan </li>
```

Comment:
Are these meant to be twitter handles (in which we're good) or github usernames (in which case I'm @jennybc)?